workspace.json

```diff
@@ -3098,6 +3098,122 @@
           }
         }
       }
+    },
```

Comment:
We will need to move the extract strings script to each template lib

Reply:
done